tests/FSharp.Literate.Tests/EvalTests.fs

```diff
@@ -270,6 +263,63 @@ let ``Can include-it`` () =
   html1 |> shouldContainText "1000"
   html1 |> shouldContainText "2000"
 
+[<Test>]
+let ``Can include-fsi-output`` () =
+  let content = """
+let xxxxxxx = 1
+(*** include-fsi-output ***)
+"""
+  let fsie = getFsiEvaluator()
+  let doc1 = Literate.ParseScriptString(content, "." </> "A.fsx", formatAgent=getFormatAgent(), fsiEvaluator = fsie)
+  let html1 = Literate.ToHtml(doc1)
+  html1 |> shouldContainText "val xxxxxxx : int = 1"
+
+[<Test>]
+let ``Can include-html-output`` () =
+  let content = """
+type Html = Html of string
+#if HAS_FSI_ADDHTMLPRINTER
+fsi.AddHtmlPrinter(fun (Html h) -> seq [], h)
+#endif
+Html "<b>HELLO</b>"
+(*** include-it ***)
+"""
+  let fsie = getFsiEvaluator()
+  let doc1 = Literate.ParseScriptString(content, "." </> "A.fsx", formatAgent=getFormatAgent(), fsiEvaluator = fsie)
+  let html1 = Literate.ToHtml(doc1)
+  html1 |> shouldContainText "<b>HELLO</b>"
+
+[<Test>]
+let ``Can include-html-output-even-without-hash-if-and-print-transformer-works-for-html-too`` () =
+  let content = """
+type Html = Html of string
+type Quack = Quack of string
+fsi.AddPrintTransformer(fun (Quack h) -> box (Html h))
+fsi.AddHtmlPrinter(fun (Html h) -> seq [], "<b>QUACK</b>")
+Quack "<b>HELLO</b>"
```

Comment:
I think it should be 
```suggestion
Quack "HELLO"
```

as you wrap into `<b>` in the printer - so we want to test if printer is executed

Reply:
thanks